pd/src/components/staking.rs

```diff
@@ -130,6 +131,9 @@ impl Component for Staking {
             .set_base_rates(cur_base_rate, next_base_rate)
             .await;
 
+        // All genesis validators start in the "Active" state:
+        let state = ValidatorState::Active;
```

Comment:
nit: I think it'd be clearer if, rather than having a variable here, we just passed `ValidatorSet::Active` into `save_validator` at 165 with this comment, so that when reading the `save_validator` invocation, there's no need to refer back here